spark-bigquery-connector-common/src/main/java/com/google/cloud/spark/bigquery/SparkBigQueryConfig.java

```diff
@@ -82,6 +87,12 @@
   public static final int MIN_BUFFERED_RESPONSES_PER_STREAM = 1;
   public static final int MIN_STREAMS_PER_PARTITION = 1;
   private static final int DEFAULT_BIGQUERY_CLIENT_RETRIES = 10;
+  private static final String ARROW_COMPRESSION_CODEC_OPTION = "arrowCompressionCodec";
+  private static final ImmutableList<String> PERMITTED_ARROW_COMPRESSION_CODECS =
```

Comment:
Effectively duplicating `ArrowSerializationOptions.CompressionCodec` values. It is better to do it programmatically by transforming `ArrowSerializationOptions.CompressionCodec.values()`. In our case though, this field can be deleted (see below)

Reply:
Good call. Removed this.